happiness_app/static/happiness_app/css/teampage_about.css

```diff
@@ -0,0 +1,35 @@
+/* css styles for team page and about section */
+
+
+.container {
+    
+}
+
+
+.card {
+    background-color: #f2e1c3;
+    padding: 3px;
+}
+
+
+/* is not working. buttons are supposed to be blue! */
+.btn btn-primary {
```

Comment:
Issue (Blocking): This don't work because `btn-primary` looks for an element called `btn-primary`, but you want to select the class. To select a class, you need `.btn-primary`. Also, if you put a space between `.btn` and `.btn-primary`, it is selecting the children of the `.btn` as opposed to the button itself. You can either omit the `.btn` or you can write it as `.btn.btn-primary`.